src/video/windows/SDL_windowsvideo.c

```diff
@@ -374,7 +461,6 @@ static BOOL WIN_DeclareDPIAwareUnaware(SDL_VideoDevice *_this)
 
 static BOOL WIN_DeclareDPIAwareSystem(SDL_VideoDevice *_this)
 {
-#if !defined(SDL_PLATFORM_XBOXONE) && !defined(SDL_PLATFORM_XBOXSERIES)
```

Comment:
Is this platform check removal here correct ?

Reply:
Seems just fixed by @flibitijibibo with https://github.com/libsdl-org/SDL/commit/c5749f0ae7c8b6252dab321e78ab3b9d53deaba1